### DISCUSSION TITLE
Has anyone tried running Moby automatic tests (CLI) against Podman

### DISCUSSION THREAD

**Discussion by eriksjolund** [containers/podman#14925] [Ideas]
I don't know anything about this but I'm just curious if there has been any attempts to run integration tests from Moby
to test the Docker-compatibility of the Podman CLI?

There seems to be some code here:
https://github.com/moby/moby/tree/master/integration-cli

(I'm not volunteering)

**Comment by mheon** [comment]
We were just discussing this earlier today. There is a desire to do so, but it'd probably be significant work to account for known differences.
